app/javascript/packs/SignUpPage.js

```diff
@@ -0,0 +1,11 @@
+import {SignupPageWrapper} from 'LoginPage'
+import {safeFromJsonString} from 'utilities/json-utils'
+
+document.addEventListener('DOMContentLoaded', () => {
+  const loginLayout = document.querySelector('.login-layout')
+  loginLayout.removeChild(document.getElementById('old-signup-page-wrapper'))
```

Comment:
In this specific case, the signup form, you might not need to do the progressive enhancement way we discussed in the past since it's an isolated case.

Reply:
It's needed for the cucumbers, or we'd need to add a lot of @javascript tags

Reply:
That's not what I meant. Adding `@javascript` tags are a drawback when the feature not necessarily needs it. However, In this case, the register step might not be needed in a lot of features, do you know how many are?. You might want to create a new cuke for testing the new register (with `@javascript` tag) OR (in the case there are not many features using register) remove the old form and add the `@javascript` tag. We shouldn't aim to trick the cukes, it's more about understand the feature we are refactoring/creating and how it's been tested... then we could either adapt, indulge or write new tests.